Makefile

```diff
@@ -39,19 +39,21 @@ vet:
 	@echo ">> vetting code"
 	@$(GO) vet $(pkgs)
 
-build: promu
+build: $(PROMU)
 	@echo ">> building binaries"
 	@$(PROMU) build --prefix $(PREFIX)
 
-tarball: promu
+tarball: $(PROMU)
 	@echo ">> building release tarball"
 	@$(PROMU) tarball --prefix $(PREFIX) $(BIN_DIR)
 
 docker:
 	@echo ">> building docker image"
 	@docker build -t "$(DOCKER_IMAGE_NAME):$(DOCKER_IMAGE_TAG)" .
 
-promu:
+promu: $(GOPATH)/bin/promu
+
+$(GOPATH)/bin/promu:
```

Comment:
Also by adding `$(GOPATH)/bin/promu` to the `.PHONY` target, we can keep the auto-update requirement when the `PROMU` var is not overriden.